pr2eus/robot-interface.l

```diff
@@ -743,7 +743,7 @@ Return value is a list of interpolatingp for all controllers, so (null (some #'i
      (:worldcoords
       (send *tfl* :lookup-transform (or (cadr args) "/map") "/base_footprint" (ros::time)))
      (:gripper
-      (send self :gripper (cdr args)))
+      (apply #'send self :gripper (cdr args)))
```

Comment:
you can use 'send *'
http://euslisp.github.io/EusLisp/manual-node33.html#4010
https://github.com/euslisp/EusLisp/blob/74fff34afa261db392800f1f2ea7518ee4cadfff/lisp/l/common.l#L688